packages/react-positioning/src/usePopper.ts

```diff
@@ -24,6 +24,26 @@ interface PopperOptions extends PositioningProps {
   enabled?: boolean;
 
   onStateUpdate?: (state: Partial<PopperJs.State>) => void;
+
+  /**
+   * Enables the Popper box to position itself in 'fixed' mode (default value is position: 'absolute')
+   * @default false
+   */
+  positionFixed?: boolean;
```

Comment:
This option was only used in v0 to break the stacking context, all v9 components currently render out of DOM elements by default, so do not expose this until needed.